community/postsrsd/APKBUILD

```diff
@@ -43,10 +44,13 @@ package() {
 
 md5sums="cb6e13a06d6bbd7d383efb9bbf0867ce  postsrsd-1.4.tar.gz
 a4104dbe7c6bc88de6a1fa29ee82e531  10-fix-defaults.patch
+bf9a65dcd4ca8b3404d71af3617208f0  20-set-socket-listen-address.patch
 575d5c5563def2aabdd752abc54b5e45  postsrsd.initd"
 sha256sums="f3230d57c36ba4688cc3beb90fbb0f199a06381e4df2edbc4ef541a9e8a574ff  postsrsd-1.4.tar.gz
 4e8093e8a90f499402f17ea9be253c499f47218bc7918d58825f6a1e2daa2543  10-fix-defaults.patch
+2e26e7299ec51a2932090676337db61f57a56dfe5a3c968440ae46100ff443ab  20-set-socket-listen-address.patch
```

Comment:
```
md5sums="cb6e13a06d6bbd7d383efb9bbf0867ce  postsrsd-1.4.tar.gz
 a4104dbe7c6bc88de6a1fa29ee82e531  10-fix-defaults.patch
+bf9a65dcd4ca8b3404d71af3617208f0  20-set-socket-listen-address.patch
 575d5c5563def2aabdd752abc54b5e45  postsrsd.initd"
 sha256sums="f3230d57c36ba4688cc3beb90fbb0f199a06381e4df2edbc4ef541a9e8a574ff  postsrsd-1.4.tar.gz
 4e8093e8a90f499402f17ea9be253c499f47218bc7918d58825f6a1e2daa2543  10-fix-defaults.patch
+2e26e7299ec51a2932090676337db61f57a56dfe5a3c968440ae46100ff443ab  20-set-socket-listen-address.patch
 e59e9193998041023f6ef2740fd8ddaa2903d2a88f39e14f638c5ca2ce023b3b  postsrsd.initd"
```

The md5 and sha256 checksums can be removed, it is leagacy.

If you use abuild checksum it will automatically generate the new checksums and remove the old - https://wiki.alpinelinux.org/wiki/Creating_an_Alpine_package#Setup_your_system_and_account

Reply:
` || return 1` can also be removed.